### PR TITLE
MarketPress Integration

### PR DESCRIPTION
**Pull request by tubiz**

### CODE REVIEW

File: includes/integrations/class-marketpress.php [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+	public function add_pending_referral( $order = array() ) {
+
+		if ( $this->was_referred() ) {
+
+			if( '0' == $order->post_author ){
+
+				$customer_email = $order->mp_shipping_info[ 'email' ];
+
+			}
+			else{
```
Comment (pippinsplugins) [inline includes/integrations/class-marketpress.php]: Could you put these on the same line and place a space before `{`?

**Comment by tubiz** [comment]
What exactly should I put on the same line?

**Comment by pippinsplugins** [comment]
Looks good to me.

**Comment by pippinsplugins** [comment]
#543

**Pull request merged**